### PR TITLE
Create simpler EMIS listing view to avoid large datasets

### PR DESCRIPTION
**Pull request by imsickofmaps**

### CODE REVIEW

File: hierarchy/api.py [hunk 1]
```diff
@@ -69,3 +69,23 @@ def prepend_urls(self):
...
+
+
+class EmisResource(ModelResource):
+    """
+    This class:
+        - Adds resource_name for the API
+        - Returns the required data for the API via Foreign key association,
+        based on the url
+    """
+    # zone = fields.ForeignKey(ZoneResource, 'zone', full=True)
```
Comment (hodgestar) [inline hierarchy/api.py]: Can the commented out thing just be deleted? Or does it need to be here?

File: hierarchy/urls.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
 from django.conf.urls import patterns, url, include
-from hierarchy.api import ProvinceResource, DistrictResource, ZoneResource, SchoolResource
+from hierarchy.api import ProvinceResource, DistrictResource, ZoneResource, SchoolResource, EmisResource
```
Comment (hodgestar) [inline hierarchy/urls.py]: This probably need to be split over multiple lines -- just do `from X import (W, Y, Z)` and put line breaks and indentation inside the brackets as needed.

**Comment by hodgestar** [comment]
Some minor comments and it might be good to have some tests for the Django side, but otherwise :+1:.

**Comment by imsickofmaps** [comment]
@hodgestar applied the suggestions except more tests :)

**Comment by hodgestar** [comment]
Looks good.

**Pull request merged**